src/app/layout/components/header/header.component.html

```diff
@@ -1,4 +1,4 @@
-<header class="header">
+<header class="header" *ngIf="isLogged; else unauthorized">
   <div class="header-container">
```

Comment:
You will delete header block for unauthorised users!

Reply:
Ok, so what should i do to fix this?

Reply:
You need to move directive to the following tag:
`<div class="header-container">`
It allows to change container in header.